notebooks/building_production_ml_systems/solutions/1_training_at_scale_vertex.ipynb

```diff
@@ -8,7 +8,8 @@
     "**Learning Objectives:**\n",
```

Comment:
<p>I got an error for port 8080 even though I didn't have any other tensorboard instance using this port. To be safe, I think we can use an completely random port like 5050/6060</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>